gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -634,6 +634,12 @@ message Alert {
 
   optional SeverityLevel severity_level = 14 [default = UNKNOWN_SEVERITY];
 
+  // List of URL linking to an image to be displayed along the alert text. Used to explain visually the alert effect of a detour, station closure, etc. The image should enhance the understanding of the alert and must not be the only location of essential information.
+  // The following types of images are discouraged : image containing mainly text, marketing or branded images that add no additional information. 
+  // The image linked need to be PNG or JPG of less than 500kB. 
```

Comment:
```suggestion
  // The images linked need to be PNG or JPG of less than 500kB. 
```
Is there a reason you want to exclude other formats such as WebP?

Reply:
I'm not strongly against WebP but the fact that the iOS support is [relatively recent](https://css-tricks.com/webp-image-support-coming-to-ios-14/) I wouldn't have included it. .